js/ftx.js

```diff
@@ -1912,18 +1912,19 @@ module.exports = class ftx extends Exchange {
         const defaultMethod = this.safeString (options, 'method', 'privateDeleteOrdersOrderId');
         let method = this.safeString (params, 'method', defaultMethod);
         const type = this.safeValue (params, 'type');  // Deprecated: use params.stop instead
-        const stop = this.safeValue (params, 'stop');
+        let isTriggerOrder = undefined;
+        [ isTriggerOrder, params ] = this.isTriggerOrder (params);
         const clientOrderId = this.safeValue2 (params, 'client_order_id', 'clientOrderId');
         if (clientOrderId === undefined) {
             request['order_id'] = parseInt (id);
-            if (stop || (type === 'stop') || (type === 'trailingStop') || (type === 'takeProfit')) {
+            if (isTriggerOrder || (type === 'stop') || (type === 'trailingStop') || (type === 'takeProfit')) {
```

Comment:
hum we're repeating `(type === 'stop') || (type === 'trailingStop') || (type === 'takeProfit')` a lot too, can we DRY it?